### PR TITLE
feat(Toast): Add more functionality, optimise styles (#912)

### PR DESCRIPTION
**Pull request by essj**
Issue [#912](https://github.com/binance-chain/ui-project-management/issues/912).

- Add optional `showCloseButton` prop, default to true.
- Change widths to new UI design, add optional `size` prop to control width.
- Add dismiss functionality.
- Add more examples in stories.

image

### CODE REVIEW

File: src/components/Toast/actions/styled.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+interface Props {
+  theme: DefaultTheme;
+  hasCloseButton: boolean;
+  size: Size;
+}
+
+export const toastSize = ({ theme, hasCloseButton, size }: Props) => {
+  const width = (() => {
+    console.log(size);
```
Comment (cchungmin) [inline src/components/Toast/actions/styled.tsx]: Let's take this down

File: src/components/Toast/actions/styled.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    if (size === 'reduced') return 280;
+    return 380;
```
Comment (cchungmin) [inline src/components/Toast/actions/styled.tsx]: I guess this magic number is from the design lol
If you so maybe we could make constants for them.

File: src/components/Toast/actions/styled.tsx (hunk 1 above)
Comment (essj) [reply src/components/Toast/actions/styled.tsx]: Whoops, good catch!

File: src/components/Toast/actions/styled.tsx (hunk 2 above)
Comment (essj) [reply src/components/Toast/actions/styled.tsx]: Yeah. 😆  Changed!

**Review by cchungmin — APPROVED** [review]
All look good to me. Thanks @essj !

**Pull request merged**